XcodeServerSDK/API Routes/XcodeServer+Auth.swift

```diff
@@ -47,24 +48,26 @@ extension XcodeServer {
     - parameter success:    Indicates whether sign out was successful.
     - parameter error:      Error indicating failure of sign out.
     */
-    public final func logout(completion: (success: Bool, error: NSError?) -> ()) {
+    public typealias Completion = (Bool, Error?) -> Void
```

Comment:
I guess because of this line header docs for `logout` won't work. Please move it above.